src/phocus_linux.go

```diff
@@ -42,6 +43,12 @@ func main() {
 		os.Exit(1)
 	}
 
+	daemonTest := os.Getenv("INVOCATION_ID")
+	if daemonTest == "" {
+		failPrint("Sorry! You're not supposed to run this binary manually. It's probably already running as a daemon (CSSClient).")
```

Comment:
Won't this just print the message and exit it immediately if they double click the `exe`?

Reply:
For Linux, I assume they're not double clicking binaries.

Reply:
Ah whoops I c/p'd